docs/units/standard_units.rst

```diff
@@ -1,3 +1,5 @@
+.. _doc_standard_units:
```

Comment:
I was lazy to check if a tag name without "doc" is already taken. Note that tag name has to be unique across the whole Astropy doc, not just that page. Adding "doc" in front (albeit redundantly) decrease the chance of conflict.